src/runtime/analytics-service.ts

```diff
@@ -286,6 +286,9 @@ export class AnalyticsService {
     const meta = new AnalyticsEventMeta();
     meta.setEventOriginator(event.eventOriginator);
     meta.setIsFromUserAction(!!event.isFromUserAction);
+    if (event.configurationId) {
+      meta.setConfigurationId(event.configurationId);
```

Comment:
It seems redundant unless we have plans to log it in clearcut events. For it to be properly propagated, a conversion logic needs to be added in analytics service here:
http://google3/java/com/google/subscribewithgoogle/client/ui/common/analytics/analytics_service.ts;l=656;rcl=625751917

Reply:
I don't think we plan on gathering this information. Will remove.